src/accelerate/accelerator.py

```diff
@@ -340,6 +341,8 @@ def __init__(
         self.init_handler = None
         self.fp8_recipe_handler = None
         self.autocast_handler = None
+        self._has_lomo_optimizer = False 
```

Comment:
We don't do private, so let's make this public please!
```suggestion
        self.has_lomo_optimizer = False 
```